views/layouts/layout-example-full-page.njk

```diff
@@ -14,9 +14,7 @@
 
 {% block header %}
   {{ govukHeader({
-    homepageUrl: "#",
-    serviceName: "Service name",
-    serviceUrl: "#"
+    homepageUrl: "#"
```

Comment:
I see no issue with this, as it matches our [default, minimal page template](https://deploy-preview-4784--govuk-design-system-preview.netlify.app/styles/page-template/#default).